sample_code/ep332/rev01/t.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import berry_as_FLUFL
```

Comment:
did I typo this or you? should be barry

Reply:
my bad, I should of tried to validated this one in the console as well, because the `SyntaxError` shadows the import error, when I executed the file :D ...